spec/features/browsing_authors_spec.rb

```diff
@@ -27,8 +27,9 @@
             expect(page).to have_content authors.first.name
           end
         end
-        scenario 'I can see the zines created by a given author'
-
+        scenario 'I can see the zines created by a given author' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.